.eslintrc.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
         devDependencies: [
           './**/stories/**/*',
           './**/*.stories.@(tsx|mdx)',
-          './**/*.test.@(ts|tsx)',
+          './**/*.Test.@(ts|tsx)',
```

Comment:
Why would `test` be uppercase? I'd expect `Component.test.tsx`

Reply:
that would not be PascalCase anymore

Reply:
We don't uppercase file extension though, `test` has a very similar function in my mind

Reply:
as discussed in slack. Unfortunately not possible